Dockerfile

```
@@ -0,0 +1,31 @@
+# Install system dependencies
+FROM python:3.10-buster AS base
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
+
+# Copy across dependecy management files
+WORKDIR /todo-app
+COPY poetry.lock /todo-app
+COPY pyproject.toml /todo-app
+
+# Install package dependencies
+RUN ~/.local/share/pypoetry/venv/bin/poetry install
```

Again not important, but possibly cleaner to read to update the PATH and then just use `poetry` directly:
```suggestion
ENV PATH=/root/.local/share/pypoetry/venv/bin/:${PATH}
RUN poetry install
```